### PR TITLE
New designs

### PR DESCRIPTION
**Pull request by vercia**

### CODE REVIEW

**Comment by kryptt** [comment]
This is looking great!

File: src/data/menuItems.ts [hunk 1]
```diff
@@ -1,16 +1,24 @@
 export const menuItems = {
   iterableItems: [
     {
-      label: 'home',
-      link: '#'
+      label: 'litepaper',
+      link: 'https://docs.fida.finance/whitepaper.pdf',
```
Comment (kryptt) [inline src/data/menuItems.ts]: Correct link is "https://docs.fida.finance/introduction/whitepaper"

File: src/data/socials.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import github from 'assets/icons/socials/github.svg';
+import medium from 'assets/icons/socials/medium.svg';
+import youtube from 'assets/icons/socials/youtube.svg';
+import twitter from 'assets/icons/socials/twitter.svg';
+
+export const socials = [
+  {
+    src: discord,
+    name: 'discord',
+    link: 'https://discord.gg/BN4NJebn'
```
Comment (kryptt) [inline src/data/socials.ts]: Is this link valid? "https://discord.com/invite/kFdXXBqwdu" however does work

File: dictionary.txt [hunk 3]
```diff
@@ -353,5 +352,194 @@ isMaxHeight750
...
+imageHeight
+setImageHeight
+imageWidth
+setImageWidth
+activeId
+onImageLoad
+HTMLImageElement
+imageElement
+onLoad
+seemlessAIMobile
```
Comment (kryptt) [inline dictionary.txt]: It is actually spelled Seamless  so there's probably a typo! :D

File: dictionary.txt [hunk 4]
```diff
@@ -353,5 +352,194 @@ isMaxHeight750
...
+textPrimaryOnBrand
+textTertiary600
+buttonTertiaryColorFg
+backgroundGradient
+backgroundGradientBottom
+greyBackgroundGradient
+iconGradient
+textSecondaryHover
+utilityGray
+rose
```
Comment (kryptt) [inline dictionary.txt]: "rose" wasn't in the default dictionary?! that's wierd

File: dictionary.txt [hunk 5]
```diff
@@ -353,5 +352,194 @@ isMaxHeight750
...
+UsePhaseSizeInput
+youtube
+github
+ninefold
+minWidth640
+justifyContent
+onClickLink
+learnAbout
+WithRoseText
+amet.
```
Comment (kryptt) [inline dictionary.txt]: where was this used? feels weird and I only see it in this file

File: dictionary.txt [hunk 6]
```diff
@@ -353,5 +352,194 @@ isMaxHeight750
...
+welcomeTexts
+welcomeHeroCard
+WelcomeCards
+howItWorks
+ourMissionTexts
+missionCards
+conquerRiskTexts
+fidaMarketplaceTexts
+seemlessAITexts
```
Comment (kryptt) [inline dictionary.txt]: here it shows up again

File: src/data/texts.ts [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+    buttonLink: '#howItWorks',
+    description: 'Include Fida Cards into your portfolio to receive gains from insurance premiums.',
+    title: 'Expand your investors portfolio'
+  },
+  card2: {
+    description: 'Insurance gap',
+    title: '67%'
+  },
+  card3: {
+    description: 'Market value',
```
Comment (kryptt) [inline src/data/texts.ts]: exptected APY

16-54%

File: src/data/texts.ts [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+
+const welcomeCards = {
+  card1: {
+    buttonLabel: 'Learn more',
+    buttonLink: '#howItWorks',
+    description: 'Include Fida Cards into your portfolio to receive gains from insurance premiums.',
+    title: 'Expand your investors portfolio'
+  },
+  card2: {
+    description: 'Insurance gap',
```
Comment (kryptt) [inline src/data/texts.ts]: Launch Policies
3

**Review by kryptt — CHANGES_REQUESTED** [review]
Some details I noticed

File: dictionary.txt (hunk 5 above)
Comment (vercia) [reply dictionary.txt]: Yeah, that was in temporary "Lorem ipsum" text, my bad 😅 thanks for CR

File: src/data/texts.ts [hunk 9]
```diff
@@ -0,0 +1,145 @@
...
+    title: '16-54%'
+  }
+};
+
+// our mission section
+const ourMissionTexts = {
+  title: 'Our Mission',
+  description: 'Ensuring comprehensive coverage for global emerging risks by redefining how risk is transferred is a financial, social, and moral imperative.',
+  values: [
+    { value: '$ 10 tn/FT', label: 'Estimated gap' },
```
Comment (kryptt) [inline src/data/texts.ts]: { value: ''$ 10 tn', label: 'FT Estimated gap' }

**Review by kryptt — APPROVED** [review]

**Pull request merged**